src/fu-util-common.c

```diff
@@ -1610,9 +1611,21 @@ fu_util_security_attrs_to_string (GPtrArray *attrs)
 				FwupdSecurityAttr *attr = g_ptr_array_index (attrs, i);
 				if (!fwupd_security_attr_has_flag (attr, hpi_suffixes[j]))
 					continue;
+				if (!help &&
+				    fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE) &&
+				    !fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS))
+					help = TRUE;
 				fu_security_attr_append_str (attr, str);
 			}
 		}
 	}
+
+	if (help) {
+		/* TRANSLATORS:  this is instructions on how to improve the HSI suffix */
+		g_string_append_printf (str, "\n%s: %s",
+					_("Your system has runtime issues. To improve your security visit"),
+					"https://github.com/fwupd/fwupd/wiki/Low-host-security-ID");
```

Comment:
I don't think translators like having URLs - maybe build the string using `g_string_new()` and use %s for the URL.

Reply:
I adjusted it to model off of what was done in `fu-util.c` for a URL.